parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -130,10 +131,17 @@ configureCNI
 ensureDHCPv6
 {{end}}
 
+{{/* containerd should not be configured until cni has been configured first */}}
+{{- if NeedsContainerd}}
+ensureContainerd
+{{end}}
+
```

Comment:
is there any issue if we move the ensureDocker to here (i.e. call ensureContainerRuntime here)? Kind of weird to separate them in two places

Reply:
yea i can move that to here as well

Reply:
TODO: separate PR for cse

Reply:
addressed in cse PR